scripts/migrations/007-experimental-false.js

```diff
@@ -0,0 +1,123 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+
+const { browsers } = require('../..');
+const { walk } = require('../../utils');
+
+/**
+ * @param {object} bcd Parsed BCD object to be updated in place.
+ */
+const fixExperimental = (bcd) => {
+  for (const { path, compat } of walk(undefined, bcd)) {
+    if (!compat?.status?.experimental) {
+      continue;
+    }
+
+    // This entry is marked as experimental. Check which browsers support it.
+
+    const browserSupport = new Set();
+
+    for (const [browser, support] of Object.entries(compat.support)) {
+      // Consider only the first part of an array statement.
+      const statement = Array.isArray(support) ? support[0] : support;
+      // Ignore anything behind flag, prefix or alternative name
+      if (statement.flags || statement.prefix || statement.alternative_name) {
+        continue;
+      }
+      if (statement.version_added && !statement.version_removed) {
+        browserSupport.add(browser);
+      }
+    }
+
+    // Now check which of Blink, Gecko and WebKit support it.
+
+    const engineSupport = new Set();
+
+    for (const browser of browserSupport) {
+      const currentRelease = Object.values(browsers[browser].releases).find(
+        (r) => r.status === 'current',
+      );
+      const engine = currentRelease.engine;
+      engineSupport.add(engine);
+    }
+
+    let engineCount = 0;
+    for (const engine of ['Blink', 'Gecko', 'WebKit']) {
+      if (engineSupport.has(engine)) {
+        engineCount++;
+      }
+    }
+
+    if (engineCount > 1) {
+      compat.status.experimental = false;
+    }
```

Comment:
Could we perhaps transform this migration to go both ways, setting `true` and `false`?  

```suggestion
    compat.status.experimental = engineCount < 2;
```

Reply:
That wouldn't match our guidelines:
https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status

If we want to infer experimental status entirely by script, we also need to take age into account.

But if we think that is possible/worthwhile, then we should delete the experimental status from source and make it a build step, so not this piece of code.

Reply:
That makes sense -- in that case, this is LGTM and we can follow up in the future to add more and potentially automate the experimental status flag!